lib/hanami/hanamirc.rb

```diff
@@ -158,5 +169,17 @@ def parse_file(path)
     def path_file
       @root.join FILE_NAME
     end
+
+    # Generates a default application name based on the application directory
+    #
+    # @since 0.8.0
+    # @api private
+    #
+    # @return [String] application_name
+    #
+    # @see Hanami::Hanamirc::APPLICATION_NAME
+    def default_application_name
+      @app_name || ::File.basename(@root)
```

Comment:
@lucasallan This `@app_name` is always undefined. Where it supposed to be instantiated?